### DISCUSSION TITLE
Teredo (IPv6) 检测 & 筛选脚本 [pteredor]这个是否可以用在gotoX

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#73] [展示]
start-with-xx-net.bat、pteredor.py、win_inet_pton.py这几个个文件是否需要修改

https://github.com/XX-net/XX-Net/issues/10328

https://github.com/SeaHOH/pteredor

**Comment by SeaHOH** [comment]
只要有 Python 就能用，直接运行 pteredor.py 就行。

**Reply by SeaHOH** [reply]
pteredor 只能测试到服务器的连接性，无法测试隧道使用时的状况。

现在 IPv6 已普及，推荐使用原生。

**Comment by wpyok500** [comment]
start-with-xx-net.bat，这个文件有判断版本信息，所以蒙了。

**Comment by wpyok500** [comment]
@SeaHOH 补充几个pteredor.py里面没有的隧道

    'teredo.ginzado.ne.jp',#日本
    'win8.ipv6.microsoft.com',#(Windows RT 8.1
    'debian-miredo.progsoc.org'

**Comment by SeaHOH** [comment]
谢谢

**Comment by wpyok500** [comment]
1、win_inet_pton.py请问这个文件是干嘛用，在start-with-xx-net.bat似乎没调用。
2、pteredor.py测试后如果重置刷新间隔时间会出现未能解析服务器名的错误，不刷新间隔时间正常。
![image](https://user-images.githubusercontent.com/14154547/39555184-dd318bb0-4ea9-11e8-85e1-45a7623ee566.png)

![image](https://user-images.githubusercontent.com/14154547/39555359-6b1db934-4eab-11e8-9c81-4ec3d4a785e5.png)

**Comment by SeaHOH** [comment]
1. win_inet_pton.py 由 pteredor.py 调用。
1. 你的截图上已经显示都是 30 秒，所以不能解析不是这个引起的。
一般情况好等 10 秒就正常连接，有时要多等一会儿才能连上。

**Comment by wpyok500** [comment]
1、刚测试等了10分钟也是未能解析服务器名
2、昨天pteredor.py测试筛选后，关机或重启，也是未能解析服务器名，今天早上开电脑，查看teredo状态也是也是未能解析服务器名。
3、start-with-xx-net.bat删除这个是否会影响pteredor.py使用。
4、tart-with-xx-net.bat、pteredor.py、win_inet_pton.py我没放在xx-net底下否会影响pteredor.py使用(我自己建了个)。

**Comment by wpyok500** [comment]
自己打包的开启ipv6小工具，不知这个结构是否影响检测筛选结果。

![image](https://user-images.githubusercontent.com/14154547/39563351-5d80c208-4ee2-11e8-9180-a4128a00298d.png)

[开启ipv6小工具.zip](https://github.com/SeaHOH/GotoX/files/1969860/ipv6.zip)

**Comment by SeaHOH** [comment]
1、2 建议更换 DNS 服务器，或者使用反污染工具。

3、4 看内容就知道不影响，你又不是小白。。。
不过建议你把第一个操作中的“开启 IPv6”命令放在挑选隧道之后，不然会覆盖设置。

**Comment by wpyok500** [comment]
在开启ipv6命令中，设置默认的teredo服务器命令注释掉了。
不好意思第3条写错，应该是删除win_inet_pton.py是否会影响pteredor.py使用。
